apps/studio/src/features/editing-experience/LayoutSelection.tsx

```diff
@@ -0,0 +1,243 @@
+// Component for layout selection
+
+import React, { useState } from 'react'
+import {
+  Box,
+  Grid,
+  GridItem,
+  HStack,
+  Image,
+  Text,
+  VStack,
+  useToken,
+  Icon,
+} from '@chakra-ui/react'
+import { Button, BxRightArrowAlt } from '@opengovsg/design-system-react'
+import { BiLeftArrowAlt, BiShow } from 'react-icons/bi'
+import Preview from '~/features/editing-experience/components/Preview'
+import placeholderContent from '~/features/editing-experience/data/placeholderContent.json'
+
+interface LayoutSelectionProps {
+  pageName: string
+}
+
+type LayoutType = 'article' | 'content'
+
+const LAYOUT_DATA: {
+  layoutDisplayName: string
+  layoutTypename: LayoutType
+  layoutDescription: string
+  imageSource: string
+}[] = [
+  {
+    layoutDisplayName: 'Default',
+    layoutTypename: 'content',
+    layoutDescription: 'This is the most basic layout for your content.',
+    imageSource: '/assets/layout-card/default_layout_card.png',
+  },
+  {
+    layoutDisplayName: 'Article',
+    layoutTypename: 'article',
+    layoutDescription:
+      'Designed for the perfect reading experience. Use this layout for text-heavy content, such as news, press releases, and speeches',
+    imageSource: '/assets/layout-card/article_layout_card.png',
+  },
+]
+
+function LayoutSelection(props: LayoutSelectionProps): JSX.Element {
+  // need state here to keep track of selected layout
+  const [selectedLayout, setSelectedLayout] = useState(LAYOUT_DATA[0]!)
+  return (
+    <>
+      {/* This is the floating header */}
+      <VStack position="fixed" h="100vh">
+        <VStack p="1.5rem 2rem" w="100vw" gap="0.5rem" alignItems="flex-start">
```

Comment:
whenever i see `100vh/100vw`, i'm always abit hesitant - this forces the element to the full size of the viewport and any wrappers of this will overflow. 

does this component require such a strong guarantee? if instead what we're looking for is "full size/width of container", maybe `w="full"` (and similar) might work well here

Reply:
Yes, thanks for pointing this out. didn't consider the case where it might overflow, since initially I was approaching the layout as though it is the root component. But would prefer the safer way here